src/_gettsim/sozialversicherung/arbeitslosen/inputs.py

```diff
@@ -6,7 +6,7 @@
 
 
 @policy_input()
-def anwartschaftszeit() -> bool:
+def min_12_monate_beitragspflichtig_versichert_in_letzten_30_monaten() -> bool:
```

Comment:
Not necessary to do here, but the better option would be to use the count. More likely to be present in datasets.